### PR TITLE
fix: subtasks overflow

### LINKED ISSUE #405: Subtasks overflow without scrolling
image

### PR DESCRIPTION
**Pull request by florian-sabonchi**
closes #405

### CODE REVIEW

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
The alignments of the "Add Subtask" and "Update" buttons are different. They should be at the same level. I guess this is due to the fixed height.

Bildschirm­foto 2023-05-13 um 6 04 10 PM

File: tasks/components/layout/TaskDetailView.tsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ export const TaskDetailView = ({
           </button>
         </div>
         <div className={tw('flex flex-row flex-1 gap-x-8 mt-3')}>
-          <div className={tw('flex flex-col gap-y-4 w-[60%] min-w-[500px]')}>
+          <div className={tw('flex flex-col gap-y-6 w-[60%] min-w-[500px]')}>
```
Comment (Paul-B98) [inline tasks/components/layout/TaskDetailView.tsx:157]:
```suggestion
          <div className={tw('flex flex-col gap-y-8 w-[60%] min-w-[500px]')}>
```

I think this is the correct class

**Review by Paul-B98 — CHANGES_REQUESTED** [review]

**Review by Paul-B98 — DISMISSED** [review]

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
The layout shift of the whole scroll area is okay. But I guess that the layout shift of the update button is not intended.

https://github.com/helpwave/web/assets/40192999/99a0bde3-a94b-45d8-aa3f-9cba681332fa

File: tasks/components/SubtaskView.tsx [hunk 2]
```diff
@@ -44,12 +46,26 @@ export const SubtaskView = ({
   onChange
 }: PropsWithLanguage<SubtaskViewTranslation, SubtaskViewProps>) => {
   const translation = useTranslation(language, defaultSubtaskViewTranslation)
+  const scrollableRef = useRef<SimpleBarCore>(null)
+  const [scrollToBottomFlag, setScrollToBottom] = useState(false)
+
+  /**
+   *
+   * Automatic scrolling to the last element to give the user a visual feedback
+   */
```
Comment (DasProffi) [inline tasks/components/SubtaskView.tsx:55]:
```suggestion
   // Automatic scrolling to the last element to give the user a visual feedback
```

**Comment by MaxSchaefer** [comment]
@florian-sabonchi we should stick the button permanently to the bottom. This will prevent a layout shift.

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**